Dockerfile.rh

```diff
@@ -12,14 +12,10 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM registry.access.redhat.com/ubi8/ubi
+FROM registry.access.redhat.com/ubi8/openjdk-8
 
 MAINTAINER Sonatype <support@sonatype.com>
 
-# Optional parameters.
-ARG IQ_SERVER_VERSION=1.99.0-01
-ARG IQ_SERVER_SHA256=b8e13545503bf4b8a93a7918fc6a56e330a620c1533f91cd9ffcb4bbaaee23c5
-
 LABEL name="Nexus IQ Server image" \
   vendor=Sonatype \
   version="1.99.0-01" \
```

Comment:
Just wondering if you can use an ARG value in the LABEL so you don't have to update the version in multiple places.

Reply:
> Just wondering if you can use an ARG value in the LABEL so you don't have to update the version in multiple places.

Yeah you can https://github.com/sonatype/docker-nexus-iq-server/pull/48/commits/5e1e59d9aee5554ea119c2c9260edae516b014a7, I don't think we can change the value to get the release version though, seem to only be allowed to use a constant or an argument/environment variable